src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -723,6 +723,13 @@ configure(struct sched_cmd *cmd)
 		SCHED_DEBUG("++ configuring penguin scheduler mode\n");
 		exec->ops = &penguin_ops;
 		exec->polling_mode = cfg->polling;
+		//Interrupt may not be enabled for some of the kernel, 
+		//Need to use polling mode in that case
+		if(!cfg->cu_isr) {
+			DRM_WARN("Interrupt is not enabled for atleast one kernel."
+			   "Fall back to polling mode\n");
+		  	exec->polling_mode = 1;
```

Comment:
There are two space in before exec->poling_mode. Please use tab for indent in the driver code.